db/migrate/20160330000130_add_foreign_key_constraints.rb

```diff
@@ -0,0 +1,33 @@
+class AddForeignKeyConstraints < ActiveRecord::Migration
+  def change
+    add_foreign_key :absences, :student_school_years
```

Comment:
From looking at https://robots.thoughtbot.com/referential-integrity-with-foreign-keys, it seems we can also move `dependent: destroy` expressions here too.  I don't think we're super rigorous about that at the moment, but would be something good to try.

Also, there's a recommendation about verifying first that the current state of the production DB can satisfy these contraints. :)  We can try that out on the staging site, which should be a quick way to validate before deploying this for real.  Added https://github.com/studentinsights/studentinsights/issues/246 for future us.